docs/ERC-TO-NATIVE-WITH-REWARD.md

```diff
@@ -0,0 +1,56 @@
+## Gas Consumption `ERC-TO-NATIVE` Bridge Mode with Reward contract
+
+#### Deployment
+##### Home
+ Contract | Method | Min | Max | Avg
+----  | ---- | ---- | ---- | ----
+EternalStorageProxy|deployment|378510|378510|378510
+RewardableValidators|deployment|1615790|1615790|1615790
+EternalStorageProxy|upgradeTo|35871|30924|30913
+RewardableValidators|initialize|202711|423292|318008
+EternalStorageProxy|transferProxyOwnership|30653|30653|30653
+EternalStorageProxy|deployment|378510|378510|378510
+HomeBridgeErcToNative|deployment|5155405|5155405|5155405
```

Comment:
My understanding is that gas usage for deployment must increase - we increased the code base. Please double check,

Reply:
Got you! Thanks